python/pyspark/ml/deepspeed/deepspeed_distributor.py

```diff
@@ -127,6 +151,10 @@ def _run_training_on_pytorch_file(
         training_command = DeepspeedTorchDistributor._create_torchrun_command(
             input_params, train_path, *args
         )
+        # Spark CI doesn't have GPUs (primary use case for deepspeed); 
+        # Just make sure everything else would work
+        if TorchDistributor._E2E_MOCK:
+            return training_command
```

Comment:
Why does this need to return anything?

Reply:
old code before figuring out stuff with the SparkConf - will clean up now